Anlab.Mvc/Client/components/OrderForm.tsx

```diff
@@ -82,7 +82,7 @@ export default class OrderForm extends React.Component<undefined, IOrderState> {
         this.state = { ...initialState };
     }
     validate = () => {
-        const valid = this.state.quantity > 0 && !!this.state.project.trim();
+        const valid = this.state.quantity > 0 && this.state.quantity <= 100 && !!this.state.project.trim();

```

Comment:
@jSylvestre Yea not too interested in adding a forms lib, was just hoping there was a way of not duplicating the logic